unison-core/src/Unison/Name.hs

```diff
@@ -190,6 +275,8 @@ class Parse a b where
 instance Convert Name Text where convert = toText
 instance Convert Name [NameSegment] where convert = segments
 instance Convert NameSegment Name where convert = fromSegment
+instance Convert [NameSegment] Name where
```

Comment:
God, should we have a Convert from String to Text and back? Is there already one?

Reply:
Ha... maybe. I haven't added one. I have been pretty conservative about adding these.